hack/update-generated-bindata.sh

```diff
@@ -5,6 +5,6 @@ OUTDIR="${OUTDIR:-$PWD}"
 
 # Using "-modtime 1" to make generate target deterministic. It sets all file
 # time stamps to unix timestamp 1
-GO111MODULE=on GOFLAGS=-mod=vendor go run github.com/kevinburke/go-bindata/go-bindata -mode 420 -modtime 1 -pkg manifests -o ${OUTDIR}/pkg/manifests/bindata.go assets/...
+GO111MODULE=on GOFLAGS=-mod=vendor go run github.com/kevinburke/go-bindata/go-bindata -mode 420 -modtime 1 -pkg manifests -o ${OUTDIR}/pkg/manifests/bindata.go assets/... manifests/...
```

Comment:
Instead of `manifests/...`, can we build in only the manifests we need?

Reply:
go-bindata takes a directory list, not sure there's any real harm here other than binary size inflation?

Reply:
If specifying files individually is not an option, then another option would be to copy the desired files to a temporary directory and specify that directory instead.  But you're right, this isn't a major problem, just something that jumped out at me when I glanced at the bindata.